WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryItems/TransactionHistoryItemViewModel.cs

```diff
@@ -40,4 +40,11 @@ public class TransactionHistoryItemViewModel : HistoryItemViewModelBase
 
 		DateString = $"{Date.ToLocalTime():MM/dd/yy HH:mm}";
 	}
+
+	public override void Update(HistoryItemViewModelBase item)
+	{
```

Comment:
Can you add a safety check here for the types? As it is in `CoinJoinsHistoryItemViewModel.cs` Update method.